task-1-solution/FirstTaskSolution/app/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <!-- TextSize -->
+    <dimen name="smallTextSize">14sp</dimen>
+    <dimen name="mediumTextSize">16sp</dimen>
+    <dimen name="largeTextSize">20sp</dimen>
+    <dimen name="wideTextSize">64sp</dimen>
```

Comment:
"large'dan daha büyük" anlamına gelecek kelime bence "Wide" olmamalı. Daha uygun bir İngilizce kelime kullanabiliriz, örneğin "Huge"